app/controllers/application_controller.rb

```diff
@@ -1764,7 +1764,8 @@ def get_view(db, options = {})
 
     # Set up the grid variables for list view, with exception models below
     if !%w(Job MiqProvision MiqReportResult MiqTask).include?(view.db) &&
-       !view.db.ends_with?("Build") && !@force_no_grid_xml && (@gtl_type == "list" || @force_grid_xml)
+        !(view.db.ends_with?("Build") && view.db != "ContainerBuild") && 
```

Comment:
@simon3z  @DNNX @martinpovolny @matthewd 
I had to do this temporary fix to get it to work. Could anyone explain why do we have that ```view.db.ends_with?("Build")``` ? Is it still mandatory?